docs/configuration.md

```diff
@@ -1,3 +1,5 @@
+> **WARNING**: This config format is for configs in version 10.0 and beyond. Please read the [legacy config](#legacy-configs) section if you are using the old config format.
```

Comment:
Please mark this with a markdown/html comment: TODO 10.0 so we know to remove

Reply:
i guess you could also just follow up with a PR onto the docs staging branch

Reply:
You mean a TODO to remove this? The idea is that it will be removed in 11.0 / when legacy path is gone completely